src/log.cr

```diff
@@ -112,6 +113,11 @@
 #
 # The logs are emitted to `STDOUT` using a `Log::StdioBackend`.
 #
+# If `./bin/app` uses `Log.setup_from_env` you can tweak the logging as
```

Comment:
I'd rather say "the executable" than `./bin/app`.

Reply:
done